### PR TITLE
fix(Config): update runtime node version to be compatible with AWS

### PR DESCRIPTION
**Pull request by SashaBayan**
*Issue #, if available:*
N/A

*Description of changes:*
The settings of the Basic Example directory specify a node version that AWS Stacks no longer support. See the following screenshot:

Screen Shot 2020-08-08 at 10 28 30 AM

This change updates the node version to be compatible with AWS so that developers who use the Basic Example can get started more seamlessly. 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

*Please ensure all pull requests are made against the `develop` branch.*

It appears the `develop` branch is well behind the master and appears to have been unused for some time. For that reason, I am basing these changes against `master`. I'll be happy to change the base if that's an active request by the maintainers.

### CODE REVIEW

**Comment by yeokyeong** [comment]
please check and confirm this pull request.
This was why "basic-starter" didn't work. 
I spent lots of time because of this one:(

**Comment by georgemck** [comment]
Just had this problem, too

**Review by j0k3r — APPROVED** [review]

**Comment by SashaBayan** [comment]
@j0k3r -- thanks for the review / approval!

Unfortunately, it looks like GH still needs an approval from someone with write access:

Screen Shot 2020-12-08 at 12 20 35 PM

It looks like others are still experiencing the issue that this PR resolves. Is there another maintainer with write access whose 👍 we can get on this? 🙏

**Comment by brett-vendia** [comment]
Thanks! This will be released in v4

**Pull request closed without merging**
